I/iso_codes/build_tarballs.jl

```diff
@@ -3,40 +3,27 @@
 using BinaryBuilder
 
 name = "iso_codes"
-version = v"4.15.1"
-
-# the git tag used for versioning has changed format
-if version < v"4.8"
-    if version < v"4.5" && version.patch == 0
-        tag = "iso-codes-$(version.major).$(version.minor)"
-    else
-        tag = "iso-codes-$version"
-    end
-elseif version == v"4.15.1" # for fake patch version to fix windows install.
-    tag = "v4.15.0"
-else
-    tag = "v$version"
-end
+version = v"4.17.0"
 
 # Collection of sources required to build iso-codes
 sources = [
-    ArchiveSource("https://salsa.debian.org/iso-codes-team/iso-codes/-/archive/$tag/iso-codes-$tag.tar.bz2",
-                  "ca2cadca98ad50af6e0ee4e139ec838695f75729d7a2c6353d31d9dfc6d3f027"),
+    ArchiveSource("https://salsa.debian.org/iso-codes-team/iso-codes/-/archive/v$(version)/iso-codes-v$(version).tar.bz2",
+                  "6d178ef4cea44e55f02eb546b29dcf68d88e9f7a68e0dd7913f5465dbaf8fc14"),
 ]
 
 # Bash recipe for building across all platforms
 script = raw"""
-cd $WORKSPACE/srcdir/iso-codes-*/
+cd $WORKSPACE/srcdir/iso-codes-*
 apk update
 apk add gettext
-./configure --prefix=${prefix} --build=${MACHTYPE} --host=${target}
+./configure --prefix=${prefix}
 make -j${nproc}
 make install
 """
 
 # These are the platforms we will build for by default, unless further
 # platforms are passed in on the command line
-platforms = supported_platforms()
+platforms = [AnyPlatform()]
```

Comment:
#7926

Reply:
Admittedly, adding a comment of why that's necessary would have been helpful. CC: @nhz2.

Reply:
Do you know where the "fix windows install" is happening? There is no architecture-specific code in the build script.

Reply:
The auditor automatically replaces all symlinks with copies in tarballs for Windows (but Windows only, not AnyPlatform): https://github.com/JuliaPackaging/BinaryBuilder.jl/blob/2a5e7117be9ade099cd1092e7a444bdcffc37500/src/Auditor.jl#L261-L274.  You won't see any platform-specific code here.